RDBM/Record.cpp

```diff
@@ -0,0 +1,181 @@
+#include"Record.h"
+#include<cstdlib>
+Record::Record()
+{
+	id = ++ID;
+}
+
+Record& Record::Add(std::string value)
+{
+	char type;
+	int ival;
+	double dval;
+	int checki = 1;
+	int checkd1 = 1;
+	int checkd2 = 0;
+	for (int i = 0; i < value.size(); i++)
+	{
+		if (isdigit(value[i]))
+		{
+			checki *= 1;
+			checkd1 *= 1;
+		}
+		else if (value[i] == '.')
+		{
+			checkd2++;
+		}
+		else
+		{
+			checki = 0;
+			checkd1 = 0;
+		}
+		checki *= checki;
+		checkd1 *= checkd1;
+	}
+	
+	
+	if (checkd1&&checkd2 == 1)
```

Comment:
Very strange statement.
checkd1&&checkd2 == 1 means that checkd1 is not null, checkd2 is not null. And if both of them are not null, it will pass this check.
As I understood, you want to check if they're value is set to 1.
If so, it is better to write
1) Either if(checkd1 == 1 && checkd2 == 1
2) or if(!(checkd1 & checkd2 ^ 1))